device/config/constants.go

```diff
@@ -0,0 +1,61 @@
+package config
+
+const (
+	// Default Configurator constants that are describe a specific configuration option
+	Locale    = "Locale"
+	Keymap    = "Keymap"
```

Comment:
__[golint]__
* [golint] exported const Keymap should have comment (or a comment on this block) or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->